### PR TITLE
Reduced the font-size on 'find address' button on mobile

### PR DESCRIPTION
**Pull request by deniselashlley**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1415,8 +1406,8 @@ currently-unhandled@^0.4.1:
     array-find-index "^1.0.1"
 
 "customizr@https://github.com/Modernizr/customizr/tarball/develop":
-  version "1.0.0-alpha"
-  resolved "https://github.com/Modernizr/customizr/tarball/develop#bf72bdbc0e693e253cfc2b80180eb7b984753111"
+  version "1.0.0"
+  resolved "https://github.com/Modernizr/customizr/tarball/develop#7f45419c18d8fefc1378cd1ca00bd2aa3aa501b5"
   dependencies:
```
Comment (Heleen-cr) [inline yarn.lock]: I think the error you got was expecting the hash that's deleted here..

Comment (Heleen-cr) [reply yarn.lock]: there was a fix for it a month ago adding that hash, so not sure why it was complaining about it expecting the hash we actually added? https://github.com/comicrelief/pattern-lab/commit/0e457b1d9f451cfa6cb793509b39b46656319f08

Comment (deniselashlley) [reply yarn.lock]: ???

Comment (Heleen-cr) [reply yarn.lock]: considering it's fixing it, I'm merging

**Pull request merged**